src/hooks/profile/useCalendarData.ts

```
@@ -0,0 +1,125 @@
+  count: number; // 해당 상태의 예약 개수
+  nickname: string; // 표시용 (실제로는 사용하지 않음)
+};
+
+// 📊 대시보드 데이터 타입 (기존과 동일)
+interface ScheduleData {
+  id: number | string;
+  scheduleId?: number | string;
+  timeSlot: string;
+  startTime: string;
+  endTime: string;
+  reservations: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)